package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/checkout-sdk",
-  "version": "1.324.0",
+  "version": "1.324.1",
```

Comment:
Think you don't need to update this and include changes in the `dist` folder.
You may want to rebase this PR.

Reply:
Yeah, its better rebase it first

Reply:
Yeah, it should be rebased first

Reply:
yes, my mistake
cleanup PR from unnecessary files